kernelci/config/lab.py

```diff
@@ -84,7 +90,25 @@ def queue_timeout(self):
     def from_yaml(cls, lab, kw):
         priority = lab.get('priority')
         if priority:
-            kw['priority'] = priority
+            if priority == 'low':
+                priority = 0
+            elif priority == 'medium':
+                priority = 50
+            elif priority == 'high':
+                priority = 100
```

Comment:
We could probably just drop the old-style low/medium/high keywords and only rely on numeric values in the YAML config to simplify this.

Reply:
We could, but I didn't want to needlessly make configuration files for other deployments instabuggy on upgrade (especially with the sometimes shaky error reporting we have) and it seems helpful for talking to lab owners given that numerical priorities aren't that widely used.

The other use with dynamic scheduling is that if we use baseline test results (or other smoke test type things) to decide what else to schedule then it could be beneficial to get them in quickly so that anything we key off them gets scheduled more promptly.